---
title: "Measles virus genomic epidemiology "
keywords: protocol
layout: document
last_updated: Aug 23, 2024
tags: [protocol] 
permalink: /viruses/mev/mev-background.html
folder: mpxv
title_text: "Mpox virus sequencing"
subtitle_text: "A primer on measles virus genomic epidemiology and why we do it"
document_name: "ARTIC-MeV-guide"
version: v1.0.0
creation_date: 2024-08-20
forked_from: 
author: ARTIC team
citation: "O'Toole et al"
nav_menu: false
show_tile: false
category: mev-guide
---

## Measles genomic epi: Why do it?

### 1. Introduction
We have prepared this document to guide public health labs and researchers on how to perform WGS of measles virus for epidemiological surveillance, with an emphasis on use cases for outbreak tracking, transmission chain resolution, and global elimination goals.

For any new clusters of measles cases, the goal is to establish the extent of the outbreak and to attenuate it as quickly as possible. Genomic surveillance can be a useful tool for measles and can help inform these public health interventions in a number of ways. It can help to establish whether cases are linked, whether they are due to importations from abroad or whether there is local circulation of the virus. Within a given outbreak, genomics can be used to identify transmission chains, help to inform routes of transmission or superspreading events and hotspots of circulation. With a representative sample of genomes, it also may be possible to assess the true size of the outbreak, and whether there is undetected circulation elsewhere. 

### 2. Background on Measles Virus and Surveillance
Measles virus is a single-stranded RNA virus, with a genome ~15,894 nucleotide bases long. The known diversity of measles virus is currently split into eight clades (A-H), which are further subdivided into 24 genotypes. Since 2019, only 2 genotypes (B3 and D8) have been detected globally, as opposed to 18 co-circulating genotypes in 2003 (Bankamp et al. 2024). 


Previously, much of the molecular surveillance for measles virus has focused on sequencing a small portion of the genome, the N450 region. This region is the minimum amount of the measles virus genome that enables classification into a specific clade and genotype, and is the most variable region of the measles virus genome (Beaty and Lee 2016).


The decline in measles cases over the last few decades has led to a bottleneck in the virus population, with a reduction in observed diversity. The recent upsurge in measles cases stem from two genotypes,  which do not readily resolve themselves into informative clusters based on the N450 region alone. Rough estimations based on the general observed rate of evolution for measles virus would anticipate 0.18 observed mutations per year for the N450 region alone versus 6.4 observed substitutions per year for the entire genome. 


Sequencing the whole genome can provide better resolution between samples to inform outbreak investigations and identify imports. With the advent of the readily adopted tiled-amplicon schemes developed by the ARTIC Network, whole genome sequencing is as accessible as sub-genomic N450 sequencing. 


As of 2025-07-23, NCBI-Virus contains 1,142 measles virus genomes that have not been marked as lab-passaged or vaccine strain, and 19,101 shared N450 nucleotide records.

Measles virus is a single stranded RNA virus, however the observed evolutionary rate of the virus population has been reported as 2x10^-4 nucleotide substitutions per site, which is relatively low for an RNA virus and has been attributed to high constraint on virus genome evolution (Beaty and Lee 2016; Jenkins et al. 2002). Our estimate based on a maximum likelihood tree with a HKY substitution model based on all genomes available on Genbank (as of 2025-04-24, n=1005) gives an estimate of 4x10^-4 nucleotide substitutions per site. 